rust-runtime/aws-smithy-runtime-api/src/client/interceptors/context.rs

```diff
@@ -294,7 +294,6 @@ impl<I, O, E: Debug> InterceptorContext<I, O, E> {
         }
 
         // Otherwise, rewind to the saved request checkpoint
-        // TODO(enableNewSmithyRuntime): Also rewind the ConfigBag
```

Comment:
Has this already been taken care of?

Reply:
From talking to Zelda, it sounds like it doesn't need to be done.